src/plugins/intel_cpu/src/utils/bfloat16.hpp

```diff
@@ -20,10 +20,7 @@ namespace intel_cpu {
 
 class bfloat16_t {
 public:
-    constexpr bfloat16_t()
-        : m_value{0}
-    {
-    }
+    bfloat16_t() = default;
```

Comment:
Please member initializer then:
```cpp
uint16_t m_value{};
```

Reply:
Hi @maxnick , we have to remove overload constructor, and make sure the class bfloat16_t is trivial.
Initialization and overload constructor will conduct class change to no trivial.

Actually oneDNN also have similar bfloat16_t definition.
https://github.com/openvinotoolkit/oneDNN/blob/26ad0022000d8de10a683efb175c2acbf064e81b/src/common/bfloat16.hpp#L42

CPP trivial concept:
https://en.cppreference.com/w/cpp/types/is_trivial
https://en.cppreference.com/w/cpp/named_req/TrivialType

Reply:
Ok, got it, the member initialization violates the "trivial" class definition.